http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/Config.scala

```diff
@@ -325,7 +326,10 @@ object Config {
       SamAuthProviderConfig(
         config.getOrElse("notebookAuthCacheEnabled", true),
         config.getAs[Int]("notebookAuthCacheMaxSize").getOrElse(1000),
-        config.getAs[FiniteDuration]("notebookAuthCacheExpiryTime").getOrElse(15 minutes)
+        config.getAs[FiniteDuration]("notebookAuthCacheExpiryTime").getOrElse(15 minutes),
+        config
```

Comment:
Minor - could do `config.as[GroupName]("customAppCreationAllowedGroup")` to not get an Option?

Reply:
best I can see is `getOrElse`...guess I can do that